templates/translation-row-editor-meta-feedback.php

```diff
@@ -7,10 +7,10 @@
 		<form>
 			<h3 class="feedback-reason-title"><?php esc_html_e( 'Type (Optional)', 'glotpress' ); ?></h3>
 			<ul class="feedback-reason-list">
-				<?php
-				$reject_reasons = Helper_Translation_Discussion::get_reject_reasons();
-				foreach ( $reject_reasons as $key => $reason ) :
-					?>
+			<?php
+				$update_reasons = Helper_Translation_Discussion::get_update_reasons();
```

Comment:
Can we name this comment_reasons? It's also what you chose for display and I like it!

Reply:
I have changed the update references to comment. I think is more appropriate.